library/src/main/java/com/kizitonwose/calendarview/model/MonthConfig.kt

```diff
@@ -231,12 +227,7 @@ internal data class MonthConfig(
                 groupByWeekOfMonth
             } else {
                 // Group days by 7, first day shown on the month will be day 1.
-                val groupBySeven = mutableListOf<List<CalendarDay>>()
-                while (thisMonthDays.isNotEmpty()) {
-                    val nextRow = thisMonthDays.take(7)
-                    groupBySeven.add(nextRow)
-                    thisMonthDays.removeAll(nextRow)
-                }
+                val groupBySeven = thisMonthDays.chunked(7).toMutableList()
                 groupBySeven
```

Comment:
This can be simplified as: 
```kotlin
else {
    // Group days by 7, first day shown on the month will be day 1.
    thisMonthDays.chunked(7).toMutableList()
}
```